FinSearchUnified/BusinessLogic/Models/FindologicArticleModel.php

```diff
@@ -173,6 +173,8 @@ public function __construct(
             $this->setVariantOrdernumbers();
             $this->setProperties();
         }
+
+        $this->isCrossSellingCategoryConfigured();
```

Comment:
This should done before setting all fields.

Reply:
It will just get overridden in the `setVariant` method.